2022/en/src/K05-inadequate-logging.md

```diff
@@ -2,6 +2,8 @@
 
 A Kubernetes environment has the ability to generate logs at a variety of levels from many different components. When logs are not captured, stored, or actively monitored attackers have the ability to exploit vulnerabilities while going largely undetected. The lack of logging and monitoring also presents challenges during incident investigation and response efforts. 
 
+![Inadequate Logging - Illustration](/assets/images/K05-2022.gif)
+
```

Comment:
I'm not entirely sure what this illustration is showing. Can you please elaborate?

Reply:
It was supposed to mean the logs in read were either not recorded, or monitored. Your perspective is correct. We will improve this GIF 'slightly' to mean 'inadequate' logging and monitoring.

Reply:
The illustration is now better in representing the actual idea of _inadequate_ logging, and monitoring. Is that good?